### PR TITLE
Make updates for second edition

### PR DESCRIPTION
**Pull request by restrellado**
This PR is for edits to the readme file to bring it up to date for DSIEUR, 2nd Ed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,81 +1,95 @@
...
 
 ## Note from Our Publisher
 
 The authors of this text and the publisher Taylor and Francis are pleased to make Data Science in Education Using R available via bookdown at [datascienceineducation.com](https://datascienceineducation.com). They request that readers access the book via the website or in print form only and do not download or reproduce copies in any other form. Any attempt to do so will be considered a contravention of the publisher’s terms of availability.
 
 ## Reading the Book
 
-We wrote this book for you and are excited to share it! You can read the current version at [datascienceineducation.com](https://datascienceineducation.com). The print version is available now through [Routledge](https://www.routledge.com/Data-Science-in-Education-Using-R/Estrellado-Freer-Mostipak-Rosenberg-Velasquez/p/book/9780367422257). 
+We're excited to share this book with you! You can read the current version at [datascienceineducation.com](https://datascienceineducation.com). The print version is available now through [Routledge](https://www.routledge.com/Data-Science-in-Education-Using-R/Estrellado-Freer-Mostipak-Rosenberg-Velasquez/p/book/9780367422257).
```
Comment (ivelasq) [inline README.md:20]:
```suggestion
We're excited to share this book with you! You can read the current version at [datascienceineducation.com](https://datascienceineducation.com). The print version of the first edition is available now through [Routledge](https://www.routledge.com/Data-Science-in-Education-Using-R/Estrellado-Freer-Mostipak-Rosenberg-Velasquez/p/book/9780367422257). 
```

File: README.md [hunk 2]
```diff
@@ -1,81 +1,95 @@
...
 
-Educational data rarely comes in a “ready-to-analyze” format. As a result, it's hard for enthusiastic practitioners to feel a connection between their questions and the data needed to answer them. To get value from the data-deluge, some educational data practitioners are adopting data science tools, like R. R is an Open Source programming language for data analysis. When data science meets education, the numbers confined to websites and PDF reports are set free. Teachers, administrators, and consultants apply programming and statistics to prepare data, transform it, visualize it, and analyze it to answer questions that make a difference for their students.
+R is an Open Source programming language for data analysis. When data science meets education, practitioners can use the numbers previously confined to websites and PDF reports. Teachers, administrators, and consultants can apply programming and statistics to prepare data, transform it, visualize it, and analyze it. These practices empower practitioners to answer questions that make a difference for their students.
```
Comment (ivelasq) [inline README.md:28]:
```suggestion
R is an Open Source programming language for data analysis. When data science meets education, practitioners can use the information previously confined to websites and PDF reports. Teachers, administrators, and consultants can apply programming and statistics to prepare data, transform it, visualize it, and analyze it. These practices empower practitioners to answer questions that make a difference for their students.
```

File: README.md [hunk 3]
```diff
@@ -1,81 +1,95 @@
...
 
-Technology is transforming both the administrative and student-facing sides of education. It's becoming increasingly important for educators - not just people hired to analyze data - to understand what stories this new data tells them them about their students. Our book empowers educators from elementary school to higher education to transform educational data into actionable insights so it helps them serve their students and institutions. We wrote our book to be used as a main textbook in a graduate data science in education course. We also wrote it as a practical reference for data scientists working with education data.
+These techniques shouldn't be learned separately from education use cases. We propose learning about data science through field-specific examples. Using common field-specific language is important for learning techniques that are practical for the job. We feel that discussing data science using education-specific scenarios will make learning more fun and meaningful.
```
Comment (ivelasq) [inline README.md:32]: There's a lot of "-specific" words here. Could we vary it a bit?

File: README.md [hunk 4]
```diff
@@ -1,81 +1,95 @@
...
+
+Technology is transforming education for administrators, staff, and students. It is increasingly important for educators -- not just data analysts -- to use data to reveal the stories of their students. Our book empowers educators from elementary school to higher education to transform educational data into actionable insights. We wrote our book as a main textbook in graduate data science in education courses. We also wrote it as a practical reference for data scientists working with education data.
```
Comment (ivelasq) [inline README.md:34]: Did we write it as a textbook?

Comment (ivelasq) [inline README.md:34]:
```suggestion
Technology is transforming education for administrators, staff, and students. It is increasingly important for educators -- not just data analysts -- to use data to reveal the stories of their students. Our book empowers educators from elementary school to higher education to transform educational data into actionable insights. We wrote our book as a main textbook in graduate data science in education courses. We also wrote it as a practical reference for data practitioners working with education data.
```

File: README.md [hunk 5]
```diff
@@ -1,81 +1,95 @@
...
 
 * The diversity of data analysis skills and applications in the education field 
-* Special considerations that come with analyzing education data
-* That good data analysis has a basic workflow 
-* The wonderful opportunity we have to shape the usefulness of data science in our education jobs
 
-And, the reader will be able to:
+* Unique considerations for analyzing education data 
+
+* Using effective analysis workflows
```
Comment (ivelasq) [inline README.md:42]:
```suggestion
* How to run effective analysis workflows 
```

File: README.md [hunk 6]
```diff
@@ -85,44 +99,44 @@ And, the reader will be able to:
...
 
-Community members can contribute by making changes through a pull request. We encourage community members to do their pull requests on separate branches. This helps us keep all the changes synced up. 
+Community members can contribute by making changes through a pull request. We encourage community members to do their pull requests on separate branches. This helps us coordinate changes: 
 
- - [How to create an issue](https://help.github.com/en/github/managing-your-work-on-github/creating-an-issue) 
+* [How to create an issue](https://help.github.com/en/github/managing-your-work-on-github/creating-an-issue) 
 
- - [How to do a pull request on a separate branch](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
+* [How to do a pull request on a separate branch](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
```
Comment (ivelasq) [inline README.md:106]:
```suggestion
* [How to make a pull request on a separate branch](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
```

**Review by ivelasq — APPROVED** [review]
Looks great, @restrellado ! Just left a few comments and questions.

File: README.md (hunk 4 above)
Comment (restrellado) [reply README.md]: One of the use cases we imagined was professors using the book in a graduate school class. Leaving it as is, but no strong feelings about that. Happy to change it if others feel differently. + @joshuarosenberg and @efreer20

File: README.md (hunk 3 above)
Comment (restrellado) [reply README.md]: Fixed.

**Pull request merged**